docs/guide/tensorboard.rst

```diff
@@ -225,3 +225,38 @@ Here is an example of how to render an episode and log the resulting video to Te
     model = A2C("MlpPolicy", "CartPole-v1", tensorboard_log="runs/", verbose=1)
     video_recorder = VideoRecorderCallback(gym.make("CartPole-v1"), render_freq=5000)
     model.learn(total_timesteps=int(5e4), callback=video_recorder)
+
+
+Directly Accessing The Summary Writer
+-------------------------------------
+
+If you would like to log arbitrary data (in one of the formats supported by torch.utils.tensorboard), you 
```

Comment:
maybe a link to https://pytorch.org/docs/stable/tensorboard.html would be better?

Reply:
Good point. Added it in ffacc5f8af4c77cfea7b1b9c24ecbd7bb3bbbc4d